openshift/README.md

```diff
@@ -17,7 +17,10 @@ Follow these steps to build and deploy RadarGun image in OpenShift and run perfo
 
 2) export DOCKER_REGISTRY address where RadarGun images will be pushed for consumption by OpenShift deployments.
 
-    `export DOCKER_REGISTRY=<OpenShift's internal docker registry address>`
+```
+oc get routes -n mirror
+export DOCKER_REGISTRY=<OpenShift's internal docker registry address>
```

Comment:
Or your own namespace on Quay or any other repository, doesn't really need to be OpenShift internal repo